bin/make-consensus.py

```diff
@@ -96,6 +106,15 @@ def main():
               file=sys.stderr)
         sys.exit(0)
 
+    if args.ivar and not args.bam:
+        print('If --ivar is used, --bam must be too.', file=sys.stderr)
+        sys.exit(0)
```

Comment:
If there's an error, exit non-zero.  Then things like make and shell scripts (running with `set -e`) can detect that something went wrong and abort.

Reply:
Then why does it use `sys.exit(0)` above?

Reply:
Changed to 1.